apps/khix/src/app/globals.css

```
@@ -655,7 +703,9 @@ body {
   left: 50%;
   z-index: 5;
   color: rgba(255, 253, 241, 0.72);
-  font-family: var(--font-geist-mono), monospace;
+  font-family:
+    ui-monospace, "SFMono-Regular", "Cascadia Code", "Liberation Mono", Menlo,
```

Stylelint (17.12.0)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)